city_scrapers/spiders/det_employment_solutions_corp.py

```diff
@@ -0,0 +1,183 @@
+            try:
+                datetime.strptime(
+                    "{} {} {}".format(date_str, year_str, '10:30 am'), "%B %d %Y %I:%M %p"
+                )
+            except ValueError:
+                date_str = date_str.replace(" ", "", 1)
+            time_str = time_tuples[i]
+            start_time = time_str[0]
+            end_time = time_str[1]
+            self.meeting_dates.append({
+                'start': self._parse_datetime(start_time, date_str, year_str),
+                'end': self._parse_datetime(end_time, date_str, year_str)
+            })
+
+    def _parse_documents(self, response):
+        """Parse agenda and minutes page"""
+        link_map = self._parse_link_map(response)
+        title_map = self._parse_title_map(response)
+        for date in self.meeting_dates:
+            title_in = self._parse_title(date, title_map)
+            meeting = Meeting(
+                title=title_in,
+                description="",
+                classification=self._parse_classification(title_in),
+                start=date['start'],
+                end=date['end'],
+                all_day=False,
+                time_notes="",
+                location=self.location,
+                links=link_map[(date['start'].month, date['start'].year)],
+                source=self.start_urls[0],
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, date, title_map):
+        """Generate title from title map or return generic title"""
+        if title_map[(date['start'].month, date['start'].year)]:
+            return title_map[(date['start'].month, date['start'].year)]
+        return "Detroit Employment Solutions Corporation"
+
+    def _parse_classification(self, title):
+        """Parse or generate classification from allowed options."""
+        if "Board" in title:
+            return BOARD
+
+        if "Committee" in title:
+            return COMMITTEE
+
+        return NOT_CLASSIFIED
+
+    def _parse_datetime(self, time_str, date_str, year_str):
+        """Parse start datetime as a naive datetime object."""
+        return datetime.strptime(
+            "{} {} {}".format(date_str, year_str, time_str), "%B %d %Y %I:%M %p"
+        )
+
+    def _parse_link_map(self, response):
+        """Parse or generate links. Returns a dictionary of month, year tuples and link lists"""
+        link_map = defaultdict(list)
+
+        title_divs = response.css('div[class="meeting-min_inner-wrapper"]')
+        for div in title_divs:
+            date = div.css("p::text").extract_first().strip()
+            date = date.split('/')
+            link_start = datetime(int(date[2]), int(date[0]), int(date[1]))
+            link = div.css("a").xpath('@href').get()
+            print(link)
+            link_map[(link_start.month, link_start.year)].append({"title": "Minutes", "href": link})
```

Comment:
This would need to include the title of the meeting as the key as well as the date. Right now it's including links for the committees for board meetings as well as for the committee meetings